test/middleware_test.coffee

```diff
@@ -337,7 +349,29 @@ describe 'Middleware', ->
             )
             testDone()
 
+    describe 'receive middleware context', ->
+      beforeEach ->
+        @robot.receiveMiddleware (context, next, done) =>
+          @middleware.call @, context, next, done
+
+      describe 'response', ->
+        it 'is a match-less Response object', (testDone) ->
+          @robot.receive @testMessage, () =>
+            expect(@middleware).to.have.been.calledWithMatch(
+              sinon.match.has('response',
+                sinon.match.instanceOf(Response).and(
+                  sinon.match.has('message',
+                    sinon.match.same(@testMessage)))) # context
+              sinon.match.any                         # next
+              sinon.match.any                         # done
+            )
+            testDone()
+
     describe 'next', ->
+      beforeEach ->
+        @robot.listenerMiddleware (context, next, done) =>
```

Comment:
Do we care that there is no test coverage of `next`/`done` for receive middleware? I *think* it might be okay, but my brain isn't 100% here today.